tools/platyPS/ValidateHelp.ps1

```diff
@@ -0,0 +1,179 @@
+﻿[CmdletBinding()]
```

Comment:
Let's make this part of the same module.  Identical comments about parameters and the need to support storage and service management modules.